src/components/idleMembers/card/index.tsx

```diff
@@ -10,13 +10,15 @@ type Props = {
 }
 
 const Card: FC<Props> = ({ idleMemberUserName }) => {
+  const [isError, setIsError] = useState(false);
```

Comment:
we can use `isImageAvailable, setIsImageAvailable` here, instead of `error, isError`

Reply:
yeah sure will do that